### DISCUSSION TITLE
service does not show properly who is unfollowed / unstargazed

### DISCUSSION THREAD

**Discussion by andry81** [yhype/yhype-feedback#12] [General]
The service shows who is unfollowed:

https://yhype.me/dashboard

* Jerzy @mxmnk

But there is no such a record in the followers page:

https://yhype.me/github/profile-followers

The same with the unstargazed:

https://yhype.me/github/accounts/andry81-devops/github-accum-stats/stars

**Comment by antonkomarev** [comment]
Yes, those pages displays only the actual followers/stargazers list. That's done to not overcomplicate the interface.

**Reply by andry81** [reply]
ok, thx
